docs/src/components/home/ProductsSwitcher.tsx

```diff
@@ -0,0 +1,235 @@
+import SvgProductTemplates from 'docs/src/icons/SvgProductTemplates';
+import SvgProductDesign from 'docs/src/icons/SvgProductDesign';
+import SvgMuiX from 'docs/src/icons/SvgMuiX';
+
+import KeyboardArrowRightRounded from '@material-ui/icons/KeyboardArrowRightRounded';
+import ROUTES from 'docs/src/route';
+
+function ProductItem({
+  icon,
+  name,
+  description,
+  href,
+}: {
+  icon: React.ReactNode;
+  name: React.ReactNode;
+  description: React.ReactNode;
+  href: string;
+}) {
+  return (
+    <Box
+      sx={{
+        display: 'flex',
+        alignItems: { md: 'center' },
+        p: 2,
+        flexDirection: { xs: 'column', md: 'row' },
+      }}
+    >
+      <Box sx={{ mr: 2 }}>{icon}</Box>
+      <Box>
+        <Typography color="text.primary" variant="body2" fontWeight="bold">
+          {name}
+        </Typography>
+        <Typography color="text.secondary" variant="body2" fontWeight="regular">
+          {description}
+        </Typography>
+        <NextLink href={href} passHref>
+          <Link
+            href={href}
+            color="primary"
+            variant="body2"
+            fontWeight="bold"
+            sx={{
+              display: 'inline-flex',
+              alignItems: 'center',
+              minHeight: 24,
+              '& > svg': { transition: '0.2s' },
+              '&:hover > svg': { transform: 'translateX(4px)' },
+            }}
+            onClick={(event) => {
+              event.stopPropagation();
+            }}
+          >
+            <span>Learn more</span>{' '}
+            <KeyboardArrowRightRounded fontSize="small" sx={{ mt: '1px', ml: '2px' }} />
+          </Link>
+        </NextLink>
+      </Box>
+    </Box>
+  );
+}
+
+function Highlight({
+  children,
+  selected = false,
+  onClick,
+  sx,
+}: {
+  children: React.ReactNode;
+  selected?: boolean;
+  onClick?: BoxProps['onClick'];
+  sx?: BoxProps['sx'];
+}) {
+  return (
+    <Box
+      role="button"
+      onClick={onClick}
+      sx={{
+        borderRadius: 1,
+        transition: '0.3s',
+        height: '100%',
+        border: '1px solid',
+        ...(selected && {
+          bgcolor: (theme) =>
+            theme.palette.mode === 'dark' ? 'primaryDark.700' : 'background.paper',
+          borderColor: (theme) => (theme.palette.mode === 'dark' ? 'primaryDark.400' : 'grey.200'),
+        }),
+        ...(!selected && {
+          '&:hover': {
+            bgcolor: (theme) => (theme.palette.mode === 'dark' ? 'primaryDark.800' : 'grey.100'),
+            '@media (hover: none)': {
+              bgcolor: 'transparent',
+            },
+          },
+          borderColor: 'transparent',
+        }),
+        ...sx,
+      }}
+    >
+      {children}
+    </Box>
+  );
+}
+
+const ProductsSwitcher = ({
+  productIndex,
+  setProductIndex,
+}: {
+  productIndex: number;
+  setProductIndex: React.Dispatch<React.SetStateAction<number>>;
+}) => {
+  const productElements = [
+    <ProductItem
```

Comment:
I think it's missing a little bit of spacing between the product title, description, and the Learn more link.

Reply:
I tried but I think it create more (40px) of the overall height and make the whole section cropped.

![Uploading Screen Shot 2564-08-04 at 11.16.51.png…]()

Reply:
Is it a bad thing, though? This spacing makes it look clearer to me.

Reply:
Added, we can revisit again if someone has strong opinion on this.